CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## v0.16 (unreleased)
+
+- [#216](https://github.com/awslabs/amazon-s3-find-and-forget/pull/216): Fix for
+  a bug affecting Parquet files with complex data types as column identifier
+  generating a
+  `Apache Arrow processing error: Mix of struct and list types not yet supported`
+  exception during the Forget phase
+- [#216](https://github.com/awslabs/amazon-s3-find-and-forget/pull/216): Fix for
```

Comment:
same PR number here?

Reply:
Yes, they are both here, but happy to split it if you think it makes more sense

Reply:
ah ok it confused me because I'm used to each PR relating to a distinct fix or feature in the changelog. I think it's OK for this one but in general the PRs should be scoped to a single fix I think?